csharp/test/IceDiscovery/simple/msbuild/client/client.csproj

```diff
@@ -23,5 +23,6 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="../../../../../src/Ice/Ice.csproj" />
+    <ProjectReference Include="../../../../../src/IceDiscovery/IceDiscovery.csproj" />
```

Comment:
This ensures the plug-in is available in the test output directory.

This is required because I updated the build to use the default output  path. We previously override to use the top level `lib` and `bin` directories.